tox.ini

```diff
@@ -8,7 +8,7 @@ setenv =
 commands = coverage run {envbindir}/pytest []
 
 deps =	
-    -r{toxinidir}/requirements_dev.txt
+    -r{toxinidir}/requirements/dev.txt
```

Comment:
Did you try running tox? This will now _only_ install the dev requirements and not the prod requirements.

Reply:
I can open a PR to fix this. I think this should just be `-r{toxinidir}/requirements_dev.txt` correct?

Reply:
Yeah -- that's what it was before and it doesn't need to change.